src/lib/components/Header.svelte

```diff
@@ -2,11 +2,12 @@
   import Toolbar from "$lib/components/Toolbar.svelte";
   import MenuButton from "$lib/components/MenuButton.svelte";
   import Back from "$lib/components/Back.svelte";
+  import { layoutContentTopHidden } from "$lib/stores/layout.store";
 
   export let back = false;
 </script>
 
-<header data-tid="header-component">
+<header data-tid="header-component" class:is-hidden={$layoutContentTopHidden}>
```

Comment:
opiniated probably: I would just call it `hidden`, it's a design style class, not a logical element.